src/js/pages/OffersList/index.jsx

```diff
@@ -122,6 +136,12 @@ class OffersList extends Component {
       hasFilter = true;
       filteredOffers = filteredOffers.filter(offer => offer.paymentMethods.includes(parseInt(this.state.paymentMethodFilter, 10)));
     }
+    if (this.state.commFilter !== '') {
+      hasFilter = true;
+      filteredOffers = filteredOffers.filter(offer => {
+        return offer.user.contactData.split(':')[0] === this.state.commFilter;
```

Comment:
`return stringToContact(offer.user.contactData).method === this.state.commFilter`